app/models/documents_request.rb

```diff
@@ -2,18 +2,21 @@
 #
 # Table name: documents_requests
 #
-#  id           :bigint           not null, primary key
-#  completed_at :datetime
-#  created_at   :datetime         not null
-#  updated_at   :datetime         not null
-#  intake_id    :bigint
+#  id                       :bigint           not null, primary key
+#  completed_at             :datetime
+#  created_at               :datetime         not null
+#  updated_at               :datetime         not null
+#  archived_intakes_2021_id :bigint
```

Comment:
i still don't see the point of having "archived" in the name but i'm just about ready to stop feeding that fed horse